src/CacheManager.Serialization.Bond/project.json

```diff
@@ -0,0 +1,53 @@
+{
+  "version": "0.9.0-*",
+  "description": "CacheManager extension package providing Bond serialization for distributed caches. CacheManager is an open source caching abstraction layer for .NET written in C#. It supports various cache providers and implements many advanced features, the Bond serializer can be used in place of the default Binary Serializer",
+  "packOptions": {
+    "licenseUrl": "https://github.com/MichaCo/CacheManager/blob/master/LICENSE",
+    "projectUrl": "http://www.cachemanager.net",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/MichaCo/CacheManager"
+    },
+    "summary": "CacheManager extension package providing Bond serialization for distributed caches.",
+    "tags": [ "Caching", "Cache", "CacheManager", "Distributed Cache", "StackExchange Redis", "Azure AppFabric", "Memcached" ]
+  },
+  "authors": [ "loneshark99" ],
+  "buildOptions": {
+    "allowUnsafe": false,
+    "languageVersion": "csharp6",
+    "warningsAsErrors": true
+  },
+  "copyright": "Copyright (c) 2015 MichaConrad",
+  "configurations": {
+    "Debug": {
+      "buildOptions": {
+        "define": [ "DEBUG" ]
+      }
+    },
+    "Release": {
+      "buildOptions": {
+        "define": [ "RELEASE" ],
+        "optimize": true
+      }
+    }
+  },
+  "dependencies": {
+    "Bond.CSharp": "4.2.1",
+    "CacheManager.Core": "0.9.0-*",
+    "NETStandard.Library": "1.6.0"
```

Comment:
please do not add the NETStandard.Library to global dependencies. This would install all the mess if you have a normal .net45x project for example.